lib/manager/game_teams_manager.rb

```diff
@@ -86,4 +85,74 @@ def calc_coach_percentage(coach_stats)
   def calc_percentage(numerator, denominator)
     (numerator.to_f / denominator).round(2)
   end
+
+  def most_accurate_team(season)
+    accurate_team(season, "most")
+  end
+
+  def least_accurate_team(season)
+    accurate_team(season, "least")
+  end
+  
+  def accurate_team(season, status)
+    season = season.to_s
+    seasonal_games = game_team_by_season(season)
+    team_stats = get_game_stats(season, seasonal_games)
+
+    if status == "most"
+      accurate_team_id = team_stats.max_by do |team_id, stats|
+        stats[:goals].to_f / stats[:shots]
+      end[0]
+    elsif status == "least"
+      accurate_team_id = team_stats.min_by do |team_id, stats|
+        stats[:goals].to_f / stats[:shots]
+      end[0]
+    end
+
+    team_name = parent.get_team_name(accurate_team_id)
+  end
```

Comment:
Great use of helper method to break down most accurate team and least accurate team!